### PR TITLE
Add /get_sharding client API. Add appendPath() util function.

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Add /get_sharding debugging API.
- Add appendPath() util function.

#123

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -92,6 +94,28 @@ class BlockchainNode {
     return nonce;
   }
 
+  getSharding() {
+    const shardingInfo = {};
+    const shards = this.db.getValue(ChainUtil.formatPath(
+        [PredefinedDbPaths.SHARDING, PredefinedDbPaths.SHARDING_SHARD]));
+    for (let encodedPath in shards) {
+      const shardPath = ainUtil.decode(encodedPath);
+      const shardProofHashes =
```
Comment (liayoo) [inline node/index.js:103]: Is there a particular reason why we're not directly getting values at /{shardPath}/.shard/sharding_enabled and /{shardPath}/.shard/proof_hash_map/latest ?

Comment (platfowner) [reply node/index.js]: I think that's a good idea. Just updated so.
PTAL

**Comment by minsulee2** [comment]
LGTM

**Review by minsulee2 — APPROVED** [review]

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**